src/cluster.h

```diff
@@ -136,6 +136,7 @@ typedef struct clusterNode {
     long long repl_offset;      /* Last known repl offset for this node. */
     char ip[NET_IP_STR_LEN];  /* Latest known IP address of this node */
     char *hostname;           /* The known hostname for this node */
+    char *nodename;           /* The known human readable nodename for this node */
```

Comment:
Someone else made the fair comment that hostname should have been an sds, since we frequently call strlen. Probably should update this to sds.

Reply:
update it to sds as the "hostname"